src/Entity/User.php

```diff
@@ -176,6 +177,25 @@ public function setEmail(string $email): void
         $this->email = $email;
     }
 
+    public function getAvatar(): mixed
+    {
+        return $this->avatar;
+    }
+
+    public function setAvatar(mixed $avatar): void
+    {
+        $this->avatar = $avatar;
+    }
+
+    public function hasAvatar(): bool
+    {
+        $stats = fstat($this->avatar);
+        if ($stats !== false) {
+            return $stats[7] > 0;
```

Comment:
Is $stats['size'] also working? It allows developers to quicker read and understand what this function is doing.